lib/concurrent/atomic/thread_local_var.rb

```diff
@@ -1,3 +1,5 @@
+require 'concurrent/atomic/weak_key_map'
```

Comment:
ah it's nested inside the AbstractThreadLocalVar, disregard previous comment. It might be better to keep it in one file though, or moved to sub-dir `concurrent/atomic/thread_local_var/weak_key_map`.